.github/workflows/lint.yml

```diff
@@ -8,16 +8,12 @@ jobs:
     runs-on: macos-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Install swiftformat 
-        run: brew install swiftformat || brew upgrade swiftformat 
+      - name: Install swiftformat and swiftlint 
+        run: |
+          brew update # verifies brew knows latest versions of all CLIs
```

Comment:
This file was changed because I was seeing weird lint failures on CI that my dev machine wasn't seeing. Even after `brew upgrade ...` the CLI versions installed on the CI were not the latest. 

By adding `brew update`, we make sure that the installed CLIs are always the latest. This removed the weird CI lint failures.